README.md

```diff
@@ -27,6 +27,7 @@ To run the application you need to clone the project and run the HTML file.
 ![image](https://user-images.githubusercontent.com/72201530/193838132-4e22f252-ce18-42ef-8066-b1e392d784a8.png)
 
 ## 🔗 Links
+
```

Comment:
this should not be neccessary.

Reply:
I didnt get it the links are necessary i think 😃